test/e2e/common.go

```diff
@@ -118,6 +119,23 @@ func dumpSpecResourcesAndCleanup(ctx context.Context, specName string, clusterPr
 		LogPath:   filepath.Join(artifactFolder, "clusters", clusterProxy.GetName(), "resources"),
 	})
 
+	workloadClient := clusterProxy.GetWorkloadCluster(ctx, cluster.Namespace, cluster.Name).GetClient()
+	nodeLogPath := filepath.Join(artifactFolder, "clusters", clusterProxy.GetName(), "nodes.yaml")
```

Comment:
I'd like to keep this for future debugging, but maybe change the output folder? kind of a random place. would you prefer it with the kube-system, activity log, machine logs from the workload cluster? or with the CAPZ resources like AMCP, AMMP, which are applied to the management cluster?